__init__.py

```diff
@@ -13,11 +13,11 @@
 from bson import json_util
 
 import fiftyone as fo
-from fiftyone.core.utils import add_sys_path
 import fiftyone.operators as foo
 import fiftyone.operators.types as types
 
-
+from .voxelgpt import ask_voxelgpt_generator
+import db
```

Comment:
Isn't the idea of https://github.com/voxel51/fiftyone/pull/4029 that users should adopt this style?

```py
from voxelgpt.voxelgpt import ask_voxelgpt_generator
from voxelgpt import db
```

where the name of their source folder is treated as the package name